packages/wdio-appium-service/tests/launcher.test.js

```diff
@@ -122,7 +123,8 @@ describe('Appium launcher', () => {
             } catch (e) {
                 error = e
             }
-            const expectedError = new Error("Appium exited before timeout (exit code: 2) - Check that you don't already have a running Appium service.")
+            const expectedError = new Error('Appium exited before timeout (exit code: 2)\n' +
```

Comment:
Can we add a test that would have the error from `process.stderr.once('data', err => { error = err })`? It's the branch that makes the test failing due to missing coverage.

If it is too difficult to test this feel free to improve coverage somewhere else where it is simpler.